src/components/logo/logo.tsx

```diff
@@ -0,0 +1,64 @@
+import {
+  Heading,
+  VStack,
+  HStack,
+  Logo,
+  Link,
+} from "@nypl/design-system-react-components";
+import React from "react";
+import appConfig from "appConfig";
+
+const DCLogo = () => {
+  return (
+    <>
+      <Link isUnderlined={false} href={`${appConfig.DC_URL}`}>
```

Comment:
This link doesn't work in Vercel. It may not be defined as an environment variable.

Reply:
Do you think I need to add this as appConfig.DC_URL ? Because DC_URL is definitely in there so I'm perplexed as why these are no longer working (See more links on swim lane are also broken now)

Reply:
I think this may also need to be defined in the `next.config.js` file https://nextjs.org/docs/pages/api-reference/next-config-js/env which is very similar to how `appConfig` is structured but for nextjs to include it in the bundle.